src/iosource/pcap/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
+zeek_add_plugin(
+    Zeek
+    Pcap
+    SOURCES
+        Source.cc
+        Dumper.cc
+        Plugin.cc
+)
 
-include(ZeekPlugin)
-
-include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
-
-zeek_plugin_begin(Zeek Pcap)
-zeek_plugin_cc(Source.cc Dumper.cc Plugin.cc)
+# Treat BIFs as builtin (alternative mode).
```

Comment:
That actually breaks Zeek (i.e., btests such as `plugins.hooks` will complain). PCAP is a plugin, not a subdir-library. That's probably why I didn't want to touch the other occurrences, but for them it's fine to use the `BIFS` block since all other instances you've pointed out are actually subdir-libraries.